addons/mail/static/src/components/message/message.js

```diff
@@ -433,20 +349,18 @@ class Message extends Component {
             return;
         }
         if (this._prettyBodyRef.el && this.message.prettyBody !== this._lastPrettyBody) {
-            this._prettyBodyRef.el.innerHTML = this.message.prettyBody;
-            this._lastPrettyBody = this.message.prettyBody;
-        }
-        // Remove all readmore before if any before reinsert them with _insertReadMoreLess.
-        // This is needed because _insertReadMoreLess is working with direct DOM mutations
-        // which are not sync with Owl.
-        if (this._contentRef.el) {
-            for (const el of [...this._contentRef.el.querySelectorAll(':scope .o_Message_readMoreLess')]) {
-                el.remove();
+            this._prettyBodyRef.el.innerHTML = '';
+            if (
+                this.message.prettyBodyWithReadMoreLess &&
```

Comment:
Also when is it not existing at all?

Reply:
In case `prettyBody` is pure text content (i.e. results to text node), then `prettyBodyWithReadMoreLess` is not set (i.e. not a DOM element).